formats/generic/frontend/examples/generic_item.json

```diff
@@ -0,0 +1,15 @@
+{
+  "content_id": "170bff3e-444c-4571-b26f-259d2b8d4f27",
+  "base_path": "/any-thing",
```

Comment:
Yes, I have changed the example to an actual services and information content item.